test/specs/utils/paginator.js

```diff
@@ -0,0 +1,50 @@
+/* eslint no-multi-spaces: "off" */
+/* eslint space-in-parens: "off" */
+/* eslint brace-style: "off" */
+/* eslint max-statements-per-line: "off" */
+var _ = require('lodash');
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var Paginator = require('../../../lib/utils/paginator');
+
+describe('Paginator', function () {
+  describe('Auto height (Terminal mode)', function () {
+    var paginator;
+    var height;
+
+    beforeEach(function () {
+      process.stdout.getWindowSize = sinon.stub().returns([80, 20]);
+      paginator = new Paginator();
+      height = paginator.getHeight.bind(paginator);
```

Comment:
Call `paginator.getHeight()` directly in tests.

To make it shorter, maybe iterate over the different test case rather than creating one `it()` block for each value.